### ISSUE TITLE
Accessibility error in static guide tables

### ISSUE THREAD

**Issue by steven1046** [OpenLiberty/openliberty.io#230]
When running the Dynamic Accessibility plugin on https://openliberty.io/guides/docker.html, it gives the following error:

![image](https://user-images.githubusercontent.com/6392944/37533031-c3b3e428-290e-11e8-9b10-c1a3658edbe7.png)

This is due to a missing aria-role on this table:

![image](https://user-images.githubusercontent.com/6392944/37533047-cf8445b8-290e-11e8-9753-7afab058fea3.png)

**Comment by lavenal** [comment]
The accessibility problem is caused by missing options="header" attribute in the table attribute list. I've opened an issue against docker static guide to fix the problem.

https://github.com/OpenLiberty/guide-docker/issues/4
